src/Avalonia.X11/Dbus/LinuxMountedDriveInfoProvider.cs

```diff
@@ -0,0 +1,157 @@
+using System;
```

Comment:
It's better to have this in `Avalonia.FreeDesktop` assembly since it can be reused for wayland and fbdev/drm backends